.circleci/config.yml

```diff
@@ -403,3 +453,32 @@ workflows:
           filters:
             branches:
               only: /^build-pr-.*/
+
+      - build-android-unsigned:
+          context: mattermost-mobile-unsigned
+          requires:
+            - test
+          filters:
+            tags:
+              only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
```

Comment:
don't you want to ignore this one? why would you want to build unsigned on a tagged release?

Reply:
@Willyfrog the `unsigned` builds is what we can consider the **final** build.

During the release process we cut the *signed* beta and non-beta builds where we increment the build number every time and deploy to the test tracks in Google Play and TestFlight, any of those builds could then be promoted to be published to the stores, that means that the build published is in fact the **final** build.

Once the build is promoted then we proceed to create a [github release](https://github.com/mattermost/mattermost-mobile/releases) for the version with a tag and we manually build the *unsigned* apps to publish in the GH release. This PR makes the process to be automatic instead.

The reason we publish the *unsigned* apps is because then "anyone" can then use the build and signed them with their own certificates, usually done by customers when fusing the apps with AppDome, BlueCedar or a similar service.